tox.ini

```diff
@@ -48,18 +42,42 @@ commands =
     pre-commit run --all-files --show-diff-on-failure
     python -c 'print("hint: run {envdir}/bin/pre-commit install to add checks as pre-commit hook")'
 
+[testenv:path]
+description = verify build can run from source (bootstrap)
+skip_install = true
+setenv =
+    PYTHONPATH = {toxinidir}/src
+deps =
+    packaging>=19.0
+    pep517>=0.9.1
+    tomli>=1.0.0
+    colorama;os_name == "nt"
+    importlib-metadata>=0.22;python_version < "3.8"
+    filelock>=3
+    pytest>=6.2.4
+    pytest-cov>=2.12
+    pytest-mock>=2
+    pytest-rerunfailures>=9.1
+    pytest-xdist>=1.34
+    toml>=0.10.0
+    wheel>=0.36.0
+    setuptools>=42.0.0;python_version < "3.10"
+    setuptools>=56.0.0;python_version >= "3.10"
+commands =
```

Comment:
Can we install build and then uninstall it before running the tests instead of re-listing all of its dependencies here?

Reply:
Looking at the output from CI:

```
path run-test: commands[0] | pip uninstall -y build
Found existing installation: build 0.7.0
Not uninstalling build at d:\a\build\build\src, outside environment D:\a\build\build\.tox\path
Can't uninstall 'build'. No files were found to uninstall.
```

I wonder what the exit code from pip is and if it isn't 0 why it isn't being reinterpreted as an error.

Reply:
```
~/c/p/build (main|✔) $ PYTHONPATH=src python -m pip uninstall -y build
Found existing installation: build 0.7.0
Not uninstalling build at /[...]/build/src, outside environment /[...]/build/venv
Can't uninstall 'build'. No files were found to uninstall.
~/c/p/build (main|✔) $ echo $status
0
```

Ugh.

Reply:
I think that only works for building packages (assuming you mean the pip flag) but we could call python with `-E`.